web/src/app/monitor/services/restConnector/rest-connector.service.ts

```diff
@@ -49,7 +54,20 @@ export class RestConnectorService {
   getChartHeaders(source: ReportData): Array<string> {
     const result = new Array<string>();
     source.meta.header.forEach((header: string) => {
-      result.push(header);
+      switch (header) {
```

Comment:
same as the comment above. Client is not supposed to modify headers.

Reply:
agree